### ISSUE TITLE
ErrorCode.PoolLedgerConfigAlreadyExistsError: 306

### ISSUE THREAD

**Issue by theoturner** [OutlierVentures/ANVIL#2]
On pool setup, actors on same machine all try to create the same pool ledger config as part of their own data structure setup. 
```
_indy_loop_callback: Function returned error (<ErrorCode.PoolLedgerConfigAlreadyExistsError: 306>, 'Error: Pool ledger config already exists\n  Caused by: Pool ledger config file with name "ANVIL" already exists\n')
```
Fix: check pool exists, and only instantiate a new pool ledger config if it doesn't.

**Comment by theoturner** [comment]
Fixed in #49841b5

**Comment by AronVanAmmers** [comment]
That would be 49841b5 :) (no # for commit hashes, GitHub detects them in line)
